webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -98,8 +101,27 @@ export const Indicators = () => {
     (state: ExperimentsState) => state.tableData.hasColumns
   )
 
+  const showOnlyChanged = useSelector(
+    (state: ExperimentsState) => state.tableData.showOnlyChanged
+  )
+
   return (
     <div className={styles.tableIndicators}>
+      <CellHintTooltip
+        tooltipContent="Toggle Show Only Changed Columns"
+        delay={[1000, 0]}
+      >
+        <button
```

Comment:
Is there a reason why we're basically adding a new feature instead of building off of selected columns? Aka, "Show Only Changed" selects only changed columns and deselects none-changed columns?

Reply:
There's also a possibility of users being confused on why "selected" columns just aren't showing up in the table and assume something is broken 🤔

Reply:
> Is there a reason why we're basically adding a new feature instead of building off of selected columns? Aka, "Show Only Changed" selects only changed columns and deselects none-changed columns?

Here is a list of things to consider with the approach of building on top of selecting columns

1. User takes a long time setting up mono-repo only to display the columns related to the project they are working in. Toggles show only changed - all selection is lost.
2. What happens when show only changed is toggled off? Would we hold the previous state and return to that? Does the selection stay the same when show only changed is turned off? What happens when selection has changed when only changed is turned on?
3. Filtering can update the columns selected for show only changed. Would this also change the selection? 
4. After applying only changed I deselect a number of columns. What happens when the data updates? Do these columns reappear? Can other new changed columns appear/get selected?

> There's also a possibility of users being confused on why "selected" columns just aren't showing up in the table and assume something is broken 🤔

This is why I tried to make the indicator as obvious as possible.

Reply:
so tl;dr - it gets complicated fast. We can discuss tomorrow when we catch up.